### ISSUE TITLE
Unable to print

### ISSUE THREAD

**Issue by SgtKUSMC** [geordanr/xwing#568]
Hi there, I am unable to print any of my builds using the available print function.

Any help would be appreciated.

## YASB Bug Report

**Note:** If you are experiencing problems with the permalink URL when you have **Maul** or **Bomblet Generator** in your list, this is a known issue.  For more information please see https://www.reddit.com/r/XWingTMG/comments/7c4q6h/yet_another_squad_builder_problems_with_maul/

### Expected Behavior

State what you expect to happen here.

### Actual Behavior

State what actually happens here.

### Language

State what language translation you use.

**Comment by geordanr** [comment]
Can you provide more detail?  What happens when you click the Print button?

What browser and OS are you using?

**Comment by austingraham24** [comment]
Experiencing the same problem.
I'm on a mac (El Capitan) using Safari.

Expected Behavior:
The modal pops up and there is a print button (or other instructions) to trigger printing the content.

What actually happens:
I get the modal pop up with options. There is no button or info on how to print. If I just right click / go to file print, the page is blank.

Attempting to include screenshots:
![screen shot 2018-03-28 at 12 34 02 pm](https://user-images.githubusercontent.com/10659542/38046120-627e9be2-3284-11e8-8ab5-fc13e1a30cd9.png)

![screen shot 2018-03-28 at 12 30 31 pm](https://user-images.githubusercontent.com/10659542/38046112-5d479ac0-3284-11e8-94a2-658b2ac95fde.png)

**Comment by geordanr** [comment]
Is it happening only in Safari, or are other browsers affected?

**Comment by austingraham24** [comment]
Aha! I have more info!
So on both safari and chrome, the model pops up but since the modal is larger than what is on the screen, most of the bottom content is cut off. (Meaning, I cant keep scrolling down the page to see the rest of the modal (see image))

If I full screen them on the mac, it works fine.

**Comment by austingraham24** [comment]
Also I should mention that this is on a laptop and not a full monitor. (I haven't tested a full monitor yet)

**Comment by zavrtak** [comment]
Have you tried to press the "change" button and switch to a different printing target, preferable a printer, to actually print?
Currently your system is setup for PDF Printing, so you are "printing" directly to a PDF-File.

**Comment by austingraham24** [comment]
Yeah I was, and it did not work yesterday (remained a blank page). Today, however, something different is happening (because why not).

So now, I can right click on the modal and the print view has content inside of it. However, it does not format with the options if they are checked, it only has the black and white minimum. So it'll work, just not the full functionality (obviously in full screen to get the rest of the modal to appear and clicking print still works as it is intended)
